tokio/src/sync/mpsc/bounded.rs

```diff
@@ -603,6 +603,57 @@ impl<T> Sender<T> {
 
         Ok(Permit { chan: &self.chan })
     }
+
+    /// Try to acquire a slot in the channel without waiting for the slot to become
+    /// available.
+    ///
+    //// If the channel is full this function will return [`TrySendError`], otherwise
+    /// if there is a slot available it will return a [`Permit`] that will then allow you
+    /// to [`send`] on the channel with a guaranteed slot. This function is similar to
+    /// [`reserve`] execpt it does not await for the slot to become available.
```

Comment:
nit: `:s/execpt/except/g`

Reply:
PRS WELCOME RORY!

Reply:
If someone's going to address this typo, consider also mentioning the new `try_reserve` in `Permit`'s rustdoc (currently it only mentions `reserve` as returning it).  I can submit that PR if nobody else is on it 😄

Reply:
Go ahead.